test/test_invalid_headers.py

```diff
@@ -300,6 +311,13 @@ def test_matching_authority_host_headers(self,
         assert headers == h2.utilities.validate_headers(
             headers, hdr_validation_flags)
 
+    @pytest.mark.parametrize('hdr_validation_flags',
+                             hdr_validation_response_headers)
```

Comment:
I don't really understand your comment here. Can you clarify? I think by header validation flags you meant `hdr_validation_combos` variable, but I'm using `hdr_validataion_response_headers` which has the `is_response_header` bit set to true in all cases.